src/copyright/agent/keyword.conf

```diff
@@ -0,0 +1,20 @@
+# properties file of the fossology KEYWORD agent
+# Copyright Siemens AG 2018 
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Description: this file holds the regex configurations for the IP agent
```

Comment:
Please change to:
Description: This file holds the regex configurations for the **Keyword** agent